### ISSUE TITLE
MPEG-DASH Demuxer not available

### ISSUE THREAD

**Issue by thesofakillers** [skyzyx/homebrew-ffmpeg#1]
**TLDR**: MPEG-DASH reading was not working, demuxer missing. Removing chromaprint and explicitly enabling libxml2 resolved this issue. Happy to open a Pull Request if you want.

---

Hi. First of all thank you for this repository.

I've been trying to use it to read MPEG-DASH streams as input to FFMPEG, for example with 
```
ffplay https://dash.akamaized.net/envivio/EnvivioDash3/manifest.mpd
```
The inclusion of the line `--enable-demuxer=dash \` in the `./configure` left me optimistic but nevertheless, after the install, running 
```
ffmpeg -demuxers
```
did not list the dash demuxer, and the ffplay command above resulted in `Invalid data found when processing input`, as expected when the demuxer is missing. 

I found [this thread](https://lists.ffmpeg.org/pipermail/ffmpeg-user/2018-May/039804.html) on the ffmpeg mailing list archive which suggests that [explicitly enabling libxml2 resolves the issue](https://lists.ffmpeg.org/pipermail/ffmpeg-user/2018-May/039806.html). 

This still did not work for some reason. I then noticed that the [Homebrew-core's ffmpeg](url), which doesn't include the dash demuxer, was being installed, which may have conflicted with things. This is because it is a [dependency of chromaprint](https://github.com/Homebrew/homebrew-core/blob/2c5fd5b1157cd0dda669255a098acd3e0edba6ed/Formula/chromaprint.rb#L16), which is a [dependency of this ffmpeg-skyzyx](https://github.com/skyzyx/homebrew-ffmpeg/blob/9f26048e76ba928cb5e6da3df4968a481463ccf6/Formula/ffmpeg-skyzyx.rb#L18). See [this discussion](https://github.com/homebrew-ffmpeg/homebrew-ffmpeg/issues/13) of a similar issue in the [recommended third-party homebrew tap repository](https://github.com/homebrew-ffmpeg/homebrew-ffmpeg).

Removing the chromaprint dependency resolved this issue, and we are now able to demux MPEG-DASH streams. Of course, you are no longer able to make use of chromprint with this change. 

---

I'm happy to open a PR off of [these edits](https://github.com/thesofakillers/homebrew-ffmpeg/tree/pull-request) if you would like them in this repository.
Alternatively, if you wish to keep chromaprint and don't care about MPEG-DASH demuxing, people are welcome to use [my fork](https://github.com/thesofakillers/homebrew-ffmpeg) directly.

**Comment by skyzyx** [comment]
Hey. Just saw this yesterday. It was buried in my inbox after a vacation.

If you're cool submitting a PR, I'm happy to approve it. If you don't have time, I can poke at it over the weekend. These seem like fairly straightforward instructions.

**Comment by thesofakillers** [comment]
[Submitted](https://github.com/skyzyx/homebrew-ffmpeg/pull/2)! Cheers
